engine/src/cubos/engine/cubos.cpp

```diff
@@ -67,6 +144,10 @@ void Cubos::run()
 {
     plugins.clear();
```

Comment:
Just noticed this, why are clearing the plugins?

Reply:
We only need to keep track of plugins to ensure they're not called multiple times. At this point (run), users cannot add anything more to the engine, so the plugin list is cleared to save space